day_two_guide/topics/proc_scaling-etcd-ansible.adoc

```diff
@@ -64,8 +64,17 @@ master-2.example.com
 etcd0.example.com
 ----
 
+. If you use the service catalog, you must update its list of etcd servers:
++
+[source,bash]
+----
+$ oc edit ds apiserver -n kube-service-catalog
+----
++
+In the `--etcd-servers` parameter, add the new etcd node to the list.
+
```

Comment:
I think it's not clear enough for me, because I have not find the '--etcd-servers' parameters in below help, so could it possible add a example in doc for how to write the command line, thx:
_# oc edit ds -h
Edit a resource from the default editor 

The edit command allows you to directly edit any API resource you can retrieve via the command line tools. It will open
the editor defined by your OC _EDITOR, or EDITOR environment variables, or fall back to 'vi' for Linux or 'notepad' for
Windows. You can edit multiple objects, although changes are applied one at a time. The command accepts filenames as
well as command line arguments, although the files you point to must be previously saved versions of resources. 

The files to edit will be output in the default API version, or a version specified by --output-version. The default
format is YAML - if you would like to edit in JSON pass -o json. The flag --windows-line-endings can be used to force
Windows line endings, otherwise the default for your operating system will be used. 

In the event an error occurs while updating, a temporary file will be created on disk that contains your unapplied
changes. The most common error when updating a resource is another editor changing the resource on the server. When this
occurs, you will have to apply your changes to the newer version of the resource, or update your temporary saved copy to
include the latest resource version.

Usage:
  oc edit (RESOURCE/NAME | -f FILENAME) [options]

Examples:
  # Edit the service named 'docker-registry':
  oc edit svc/docker-registry
  
  # Edit the DeploymentConfig named 'my-deployment':
  oc edit dc/my-deployment
  
  # Use an alternative editor
  OC_EDITOR="nano" oc edit dc/my-deployment
  
  # Edit the service 'docker-registry' in JSON using the v1 API format:
  oc edit svc/docker-registry --output-version=v1 -o json

Options:
  -f, --filename=[]: Filename, directory, or URL to files to use to edit the resource
      --include-extended-apis=true: If true, include definitions of new APIs via calls to the API server. [default true]
      --include-uninitialized=false: If true, the kubectl command applies to uninitialized objects. If explicitly set to
false, this flag overrides other flags that make the kubectl commands apply to uninitialized objects, e.g., "--all".
Objects with empty metadata.initializers are regarded as initialized.
  -o, --output='yaml': Output format. One of: yaml|json.
      --output-patch=false: Output the patch if the resource is edited.
      --record=false: Record current kubectl command in the resource annotation. If set to false, do not record the
command. If set to true, record the command. If not set, default to updating the existing annotation value only if one
already exists.
  -R, --recursive=false: Process the directory used in -f, --filename recursively. Useful when you want to manage
related manifests organized within the same directory.
      --save-config=false: If true, the configuration of current object will be saved in its annotation. Otherwise, the
annotation will be unchanged. This flag is useful when you want to perform kubectl apply on this object in the future.
      --validate=false: If true, use a schema to validate the input before sending it
      --windows-line-endings=false: Defaults to the line ending native to your platform.

Use "oc options" for a list of global command-line options (applies to all commands).__

Reply:
This change looks good and complete to me.  @geliu2016 the exact command an admin must use was listed:

`oc edit ds apiserver -n kube-service-catalog`

this will bring up an editor with the contents of the daemonset for the Service Catalog API server.  If you search for "etcd" you will find the existing arguments  that specify the list of etcd servers to use.  Add the etcd urls of each etcd server, each separated by a comma.

In part, the content you are editing will look something like this:
```
Containers:
  apiserver:
    Container ID:  cri-o://6ce11a193f995be2fbf6d1ec1906cc2af343d98861c9bfdd521ae8e213233bce
    Image:         registry.reg-aws.openshift.com:443/openshift3/ose-service-catalog:v3.10
    Image ID:      registry.reg-aws.openshift.com:443/openshift3/ose-service-catalog@sha256:08a79ad42f9641b7b42a02a061116e4fa25e518724c6eb619386dd8facfa45f9
    Port:          6443/TCP
    Command:
      /usr/bin/service-catalog
    Args:
      apiserver
      --storage-type
      etcd
      --secure-port
      6443
      --etcd-servers
      https://ip-172-31-33-206.us-west-2.compute.internal:2379,https://ip-172-31-63-142.us-west-2.compute.internal:2379,https://ip-172-31-23-170.us-west-2.compute.internal:2379
      --etcd-cafile
      /etc/origin/master/master.etcd-ca.crt
      --etcd-certfile
      /etc/origin/master/master.etcd-client.crt
      --etcd-keyfile
      /etc/origin/master/master.etcd-client.key
      -v
      3
      --cors-allowed-origins
      localhost
      --enable-admission-plugins
      KubernetesNamespaceLifecycle,DefaultServicePlan,ServiceBindingsLifecycle,ServicePlanChangeValidator,BrokerAuthSarCheck
      --feature-gates
      OriginatingIdentity=true
```

Reply:
@jboyd01 Thanks for kindly explanation, LGTM.